src/app/core/interfaces/dish.ts

```diff
@@ -1,11 +1,13 @@
 export interface Dish {
+    complex: string;
     id: string;
-    name: string;
     info: string;
-    weight: string;
-    image: string;
```

Comment:
weight: *number;
*img: string;

Reply:
As I remember `weight: string` because it is possible case 200g/50g